src/rgw/rgw_redis_lock.cc

```diff
@@ -0,0 +1,40 @@
+#include "rgw_redis_lock.h"
+
+namespace rgw {
+namespace redislock {
+
+int initLock(boost::asio::io_context& io, connection* conn, config* cfg,
+             optional_yield y) {
+  return rgw::redis::loadLuaFunctions(io, conn, cfg, y);
+}
+
+int lock(connection* conn, const std::string name, const std::string cookie,
+         const int duration, optional_yield y) {
```

Comment:
though there is nothing wrong in defining a by-value parameter as `const`, the "custom" is not to do that